source/sass/component/_modal.scss

```diff
@@ -119,7 +119,7 @@ $c-modal-backdrop-color: var(--c-modal-backdrop-color, $color-alpha) !default;
     .c-modal__close.c-button.c-button--md,
     .c-modal__close.c-button.c-button--sm {
         cursor: pointer;
-        color: $color-white;
+        color: $color-black;
```

Comment:
Does this work with modals without a container. For instance, the search modal?

Reply:
This has been fixed elsewhere